bosh-deployment.yml

```diff
@@ -100,6 +100,11 @@ instance_groups:
           internal:
             hostnames:
             - (( concat "0.uaa." meta.env.name "-bosh." meta.env.name "bosh.toolingbosh" ))
+        servlet:
+          session-cookie:
+            max-age: 900
+            secure: true
```

Comment:
No reason not to do this, but http-only defaults to true, and secure defaults to require-https, which we set to true.

Reply:
👍 Just being explicit.